code/API_definitions/qos-profiles.yaml

```diff
@@ -24,17 +24,45 @@ info:
 
     It is important to remark that in cases where personal user data is processed by the API, and users can exercise their rights through mechanisms such as opt-in and/or opt-out, the use of 3-legged access tokens becomes mandatory. This measure ensures that the API remains in strict compliance with user privacy preferences and regulatory obligations, upholding the principles of transparency and user-centric data control.
 
+    # Identifying a device from the access token
+
+    This specification defines the `device` object field as optional in API requests, specifically in cases where the API is accessed using a 3-legged access token, and the device can be uniquely identified by the token. This approach simplifies API usage for API consumers by relying on the device information associated with the access token used to invoke the API.
+
+    ## Handling of device information:
+
+    ### Optional device object for 3-legged tokens:
+
+    - When using a 3-legged access token, the device associated with the access token must be considered as the device for the API request. This means that the device object is not required in the request, and if included it must identify the same device, therefore **it is recommended NOT to include it in these scenarios** to simplify the API usage and avoid additional validations.
```

Comment:
Is this optional "device" neccessarry for qos-profiles?
As there is no need of having device-identifiers for retrieving qos-profile-information

>device associated with the access token must be considered as the device for the API request.

As there is no device in any qos-profile request, this sentence can at least removed.


wdyt?

Reply:
Or are we just preparing for the future that there could be an option of processing device-data?

Reply:
It's preparation for the very near future, more specifically for #318